Editor/PackageManager/PackageDependencies/XRProviders/OculusXRPackageEnabler.cs

```diff
@@ -0,0 +1,45 @@
+﻿#if CREATOR_OCULUS
+using System;
+using UnityEngine;
+using UnityEngine.XR.Management;
+using Innoactive.Creator.Core.Utils;
+
+namespace Innoactive.CreatorEditor.PackageManager
+{
+    /// <summary>
+    /// Enables the Oculus XR Plugin.
+    /// </summary>
+    internal sealed class OculusXRPackageEnabler : Dependency, IDisposable
+    {
+        /// <inheritdoc/>
+        public override string Package { get; } = "com.unity.xr.oculus";
+
+        /// <inheritdoc/>
+        public override int Priority { get; } = 2;
+
+        public OculusXRPackageEnabler()
+        {
+            OnPackageEnabled += InitializeOculusLoader;
+        }
+
+        public void Dispose()
+        {
+            OnPackageEnabled -= InitializeOculusLoader;
+        }
+
+        private void InitializeOculusLoader(object sender, EventArgs e)
```

Comment:
code duplication, can you created a base which is handling all this reflection stuff?